src/Xamarin.Android.Build.Utilities/Sdks/AndroidSdkBase.cs

```diff
@@ -136,16 +136,24 @@ protected IEnumerable<string> FindExecutableInPath (string executable)
 		
 		protected IEnumerable<string> FindExecutableInDirectory(string executable, string dir)
 		{			
-			var pathExt = Environment.GetEnvironmentVariable ("PATHEXT");
-			var pathExts = pathExt?.Split (new char [] { Path.PathSeparator }, StringSplitOptions.RemoveEmptyEntries);
-			
 			if (File.Exists (Path.Combine (dir, (executable))))
```

Comment:
This check shouldn't be needed. We should fix `Executables()` so that it also treats `null` as an extension, which removes the extension. (Right now, it does nothing.)

That is, `Executables("ndk-stack")` should return `{ "ndk-stack", "ndk-stack.exe", "ndk-stack.cmd", ... }`, and if it *did* return those, then *this* check would be superfluous.